### PR TITLE
Translate page report template in French - fix #8918 for `fr`

### LINKED ISSUE #8918: Localized issue template (`page-report.yml`)
@JuanVqz asked if `page-report.yml` can be translated in each locale now that the new English issue template is being used on `translated-content`, so I decided to create this issue for discussion :)

I was thinking of creating a `page-report-<locale>.yml` file for each locale (e.g. `page-report-fr.yml`) in a PR, all in English first. I will update the implementation on `yari`, then locale teams can:
  - translate them on that PR with suggestions/commits
  - *or* it could be merged first, and each team creates a PR after to translate the template for their locale

Poking @mdn/yari-content-ru @mdn/yari-content-ja @mdn/yari-content-pt-br @mdn/yari-content-zh (apologies for pinging so many times 😅) who have yet not approved

**Edit:** mdn/yari#7277 should take care of the "Report on GitHub" link

### PR DESCRIPTION
**Pull request by SphinxKnight**
See #8918

### CODE REVIEW

**Comment by caugner** [comment]
Happy to merge once the changes have been approved by @cw118. 🙂

**Review by caugner — COMMENTED** [review]
Happy to merge once the changes have been approved by @cw118. 🙂

File: .github/ISSUE_TEMPLATE/page-report-fr.yml
Comment (cw118) [inline .github/ISSUE_TEMPLATE/page-report-fr.yml:52]:
```suggestion
      description: Cela peut être des étapes de reproduction du bogue, des captures ou enregistrements d'écran, un fragment de code.
```

**Review by cw118 — APPROVED** [review]
Sorry for the late review, I have only one small comment but the rest is great!

**Review by caugner — APPROVED** [review]

**Pull request merged**
